src/python/pants/vcs/git_test.py

```diff
@@ -14,8 +14,7 @@
 class GitTest(unittest.TestCase):
     @staticmethod
     def init_repo(remote_name, remote):
-        # TODO (peiyu) clean this up, use `git_util.initialize_repo`.
-        subprocess.check_call(["git", "init"])
+        subprocess.check_call(["git", "init", "--initial-branch=main"])
```

Comment:
Git prints this warning, so this has an added benefit of preventing a future CI breakage :) 

```
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint:
hint: 	git config --global init.defaultBranch <name>
hint:
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint:
hint: 	git branch -m <name>
```